packages/graphql-lowcode/src/ast/fragments/removeOrAddFragmentFields.ts

```diff
@@ -0,0 +1,141 @@
+import { BREAK, FragmentDefinitionNode, SelectionNode, FieldNode, parse, visit } from 'graphql'
+import { addFragmentField } from '../../modify'
```

Comment:
what is the different between ast and modify directory after merging this pr? (it seems that both modify the AST now) Let's move files from `../../modify` to somewhere else please.